easybuild/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb

```diff
@@ -24,6 +24,9 @@ checksums = [
     {'STAR-2.7.11a_use-external-htslib.patch': '2fdc3ed9372d983f77d861d6f16a60a553598358dce9ff8216f96eb20e63ce4e'},
 ]
 
+#by default this is set to -mavx2 which makes it fail on non x86 systems
+prebuildopts = 'CXXFLAGS_SIMD= '
```

Comment:
@jpecar This should probably be `buildopts` (so we get "`make CXXFLAGS_SIMD=`")

Reply:
Actually no. I checked build logs and prebuildopts results in the right thing (no -mavx2) while buildopts does not.

Reply:
Verified, works as intended indeed via `prebuildopts`